README.md

```diff
@@ -432,6 +433,17 @@ example `<Heading.h1>`, `<Heading.h2>`, `<Heading.h3>` etc.
 
 [View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Heading/story.js)
 
+### HighlightedName
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/HighlightedName/story.js)
+
+#### Props
+
+| Name   | Type   | Default   | Required   | Description   |
+| ------ | ------ | --------- | ---------- | ------------- |
+| `shade`        | `number` | - | - | The index for one of the pre-determined shades for the highlighted name |
```

Comment:
I think I wasn't clear when I was talking about the component, what I meant is the `Highlighted Name` component should autogenerate the color from its content, and for a start, we can only have the `children` as the only prop for the component. What I meant is, the color autogeneration feature should be very similar to what we have in the current `Badge` component (not the new `Badge` that I submitted a PR for). Let me know if anything else is unclear